recipe/migrations/arm_variant_type.yaml

```diff
@@ -0,0 +1,76 @@
+migrator_ts: 1762977869
+__migrator:
+
+  commit_message: |2
+    Build for NVIDIA Tegra devices and CUDA 12.9
+
+    This migration adds `arm_variant_type=tegra` to the build matrix to support NVIDIA Tegra
+    devices compatible with CUDA 12.9. Tegra devices compatible with CUDA 13.0 are SBSA
+    compliant, and do not need a separate build. Only Orin (sm_87) and later devices are
+    supported because earlier devices are not supported by CUDA 12.9.
+
+    The `arm-variant` package must also be added to the build requirements of the recipe in
+    order to constrain the CUDA compiler to the correct variant.
+
+    ```yaml
+    requirements:
+      build:
+        - {{ compiler('cuda') }}
+        - arm-variant * {{ arm_variant_type }}  # [linux and aarch64 and cuda_compiler_version != "None"]
+    ```
+
+    This migrator is not compatible with feedstocks still targeting CUDA 11.
+
+    Please read the conda-forge CUDA recipe guide for more information:
+    https://github.com/conda-forge/cuda-feedstock/blob/main/recipe/doc/recipe_guide.md#building-for-arm-tegra-devices
+
+  use_local: false
+  paused: true
+  kind: version
+  migration_number:
+    1
+  build_number:
+    0
+  override_cbc_keys:
+    - cuda_compiler_stub
+  check_solvable: false
+  wait_for_migrators:
+    - cuda129
+    - aarch64 and ppc64le addition
```

Comment:
not sure if this works? I don't think it makes a lot of sense to wait for that migrator in practice (even though it's required conceptually) - the aarch/ppc migration is complete up to very few packages, and if someone wants to fix up aarch support for a feedstock that doesn't have it yet because they want tegra support, all the better.